eng/pipelines/templates/steps/build-test.yml

```diff
@@ -54,6 +54,14 @@ steps:
   - ${{ if eq(parameters.TestProxy, true) }}:
     - template: /eng/common/testproxy/test-proxy-tool.yml
 
+  - task: PowerShell@2
```

Comment:
Aren't we handling this in other languages by doing it in the test framework? If this is easier for now that is fine but it would be nice if this were integrated so that it would also work for folks running the tests locally.

Reply:
@weshaggard it definitely shouldn't be necessary. Closing this PR for now.